### PR TITLE
Ana/better multiple denoms porfolio version

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

A temporary more stylish version of multiple denoms.

TODO: scroll bar is somehow missing :woman_shrugging: 

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
All the resources I have tried so far to get that scroll bar showing:

- https://iamsteve.me/blog/entry/using-flexbox-for-horizontal-scrolling-navigation
-https://stackoverflow.com/questions/57037600/how-to-create-a-horizontally-scrollable-flex-column-of-reversed-flex-rows
-https://medium.com/flexbox-and-grids/how-to-create-horizontally-scrollable-sections-with-flexbox-60d860f539b2
-https://www.pluralsight.com/guides/making-a-single-child-div-scroll
-https://stackoverflow.com/questions/21515042/scrolling-a-flexbox-with-overflowing-content

Still searching for answers... xD

**Comment by faboweb** [comment]
Good temporary fix!

**Comment by Bitcoinera** [comment]
> Good temporary fix!

It was actually Jordan's idea :wink: 

I'm just the executor

File: src/components/common/TmBalance.vue [hunk 1]
```diff
@@ -33,33 +34,31 @@
...
-          <div v-if="overview.totalRewards" class="rewards">
-            <h3>Total Rewards</h3>
-            <h2>+{{ overview.totalRewards | shortDecimals | noBlanks }}</h2>
+            <div v-if="overview.totalRewards" class="rewards">
+              <h3>Total Rewards</h3>
+              <h2>+{{ overview.totalRewards | shortDecimals | noBlanks }}</h2>
+            </div>
+          </div>
+          <div
+            v-if="balances && balances.length > 1"
```
Comment (faboweb) [inline src/components/common/TmBalance.vue]:
```suggestion
            v-if="formattedBalances.length > 0"
```

**Comment by faboweb** [comment]
@Bitcoinera did you know that you shouldn't use the same ids multiple times on one page? In this list of tokens it would have appeared multiple times

**Comment by Bitcoinera** [comment]
> @Bitcoinera did you know that you shouldn't use the same ids multiple times on one page? In this list of tokens it would have appeared multiple times

Yes, sorry about that. It was already late xD

**Pull request merged**